yeelight/flow.py

```diff
@@ -44,6 +44,10 @@ def __init__(self, count=0, action=Action.recover, transitions=None):
         self.action = action
         self.transitions = transitions
 
+        # Note, main depends on us, so we cannot import BulbException here.
+        if len(self.transitions) > 10:
+            raise Exception("More transactions (%s) than maximum of 10." % len(self.transitions))
```

Comment:
I'm not sure about this, I don't know if new firmware makes this obsolete, and I don't want to limit uses that may otherwise be fine in order to catch an implementation detail that exists now. That's why I only put it in the docs and didn't limit it in the code as well.